db/postgreSQL/create_tables.js

```diff
@@ -0,0 +1,12 @@
+const { Shoe, Review } = require('./postgres_index.js');
+
+async function synchronizeModels() {
```

Comment:
There are some options for syncing models in psql that prevent the db from being re-created or re-seeded. If that is a problem for you, try looking into those options.